datacommons/places.py

```diff
@@ -92,3 +92,59 @@ def get_places_in(dcids, place_type):
   if isinstance(dcids, pd.Series):
     return pd.Series([result[dcid] for dcid in dcids])
   return result
+
+
+def get_pop_obs(dcid):
```

Comment:
Should we move this to the Populations module? The language of get_pop_obs documentation matches more get_populations and get_observations, than it does get_places_in.

Reply:
Ah okay that's fine. This raises a couple questions for me actually.

- I think the distinction between get_place_obs and get_pop_obs is a bit confusing now. I would have assumed that observations linked directly to place would be returned by get_place_obs.
- Also how is the structure of the return value if the observation is linked directly to the place dcid (in particular, how do you differentiate Observations linked by "location" vs. "childhoodLocation"? 
- Is it possible to separate get_pop_obs into two functions with one returning population observation pairs, and the other returning observations linked directly to the Place?

Reply:
I'm also wondering if it all needs to be returned in one call. As is, the call seems very browser specific. Also, are these calls place specific? If I understand correctly, we could have statistical population on non-place nodes.

Would we run into limit issues with this call? Or is everything going to be in the cache.

Reply:
> Ah okay that's fine. This raises a couple questions for me actually.
> 
> * I think the distinction between get_place_obs and get_pop_obs is a bit confusing now. I would have assumed that observations linked directly to place would be returned by get_place_obs.
They do have things in common, kinda like get_triples and get_property_values. get_place_obs get observations for a particular population spec, while get_pop_obs get everything. We need to inform users from the comments.

> * Also how is the structure of the return value if the observation is linked directly to the place dcid (in particular, how do you differentiate Observations linked by "location" vs. "childhoodLocation"?
There is an "observations" field to contain all those. Currently there is no way to tell that, this is a bug.

> * Is it possible to separate get_pop_obs into two functions with one returning population observation pairs, and the other returning observations linked directly to the Place?

Might be easier to put them in one, this is like get_triples to get everything. get_place_obs is like get_prop_values.

Reply:
@beets, that's a good point, we do have SP linked to School etc, that said it seems reasonable to separate out get_pop_obs and get_obs. Also we should introduce "population" and "observation" modules in python.

I suggest we could discuss and do this later as it also involves mixer change.

Reply:
This is only from cache, so limit is not an issue.

Reply:
get_place_obs gives observation of places, with the population specification fixed.

get_pop_obs actually or (ultimately) get observations for a place, i would see the population here is a by-product. This is like get city in USA, the county is just a by-product.

Reply:
If this could return non-place results, should this move to the populations module? Unless the mixer currently only returns place results. Seeing the layout of functions across modules now, I think this belongs with populations.. sorry if I misunderstood your comments.

Reply:
Yea, moved to populations module

Reply:
thanks!